R/course.R

```diff
@@ -263,11 +263,12 @@ normalize_url <- function(url) {
 }
 
 conspicuous_place <- function() {
-  Filter(dir.exists, c(
-    file.path(Sys.getenv("HOME"), "Desktop"), # typical macOS = ~/Desktop
-    file.path(Sys.getenv("USERPROFILE"), "Desktop"), # typical Windows Desktop
-    "~",
-    getwd()
+  Filter(dir_exists, c(
+    path(Sys.getenv("HOME"), "Desktop"), # typical macOS = ~/Desktop
+    path_home("Desktop"), # typical Windows Desktop
+    path_home(),
+    path_home_r(),
```

Comment:
Why both `path_home()` and `path_home_r()`?

Reply:
IDK ... belt and suspenders? I agree it's hard to imagine `path_home_r()` existing and `path_home()` not existing, but I suppose it's possible if someone has fiddled with their env vars. In terms of people finding their downloaded stuff, `C:\Users\username\Documents` still seems more conspicuous than falling back to working directory.